service.subtitles.supersubtitles/service.py

```diff
@@ -0,0 +1,533 @@
+# -*- coding: utf-8 -*- 
+
+import time
+
+start = time.time()
+
+import os
+import sys
+import shutil
+import unicodedata
+import os.path
+import re
+
+import xbmc, xbmcvfs, xbmcaddon, xbmcgui, xbmcplugin
+
+import json
+
+from urllib.parse import unquote, unquote_plus, urlencode, quote_plus
+from urllib.request import urlopen, Request
+from urllib.error import HTTPError, URLError
+
+from contextlib import closing
+
+__addon__ = xbmcaddon.Addon()
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
+__cwd__ = xbmcvfs.translatePath(__addon__.getAddonInfo('path'))
+__profile__ = xbmcvfs.translatePath(__addon__.getAddonInfo('profile'))
+__temp__ = xbmcvfs.translatePath(os.path.join(__profile__, 'temp', ''))
+
+
+BASE_URL = 'http://www.feliratok.info/index.php'
+
+TAGS = [
+    'WEB-DL',
+    'PROPER',
+    'REPACK'
+]
+
+QUALITIES = [
+    'HDTV',
+    '720p',
+    '1080p',
+    'DVDRip',
+    'BRRip',
+    'BDRip',
+    'WEB',
+    'WEBRip'
+]
+
+RELEASERS = [
+    '2HD',
+    'AFG',
+    'ASAP',
+    'BiA',
+    'DIMENSION',
+    'EVOLVE',
+    'FoV',
+    'FQM',
+    'IMMERSE',
+    'KiNGS',
+    'LOL',
+    'REMARKABLE',
+    'ORENJI',
+    'TLA',
+
+    '0SEC',
+    'FLEET',
+    'KILLERS',
+
+    'AVS',
+    'BATV'
+    'SVA',
+
+    'TBS'
+]
+
+HEADERS = {'User-Agent': 'xbmc subtitle plugin'}
+
+ARCHIVE_EXTENSIONS = [
+    '.zip',
+    '.rar',
+    '.cbr'
+]
+
+LANGUAGES = {
+    "albán": "Albanian",
+    "arab": "Arabic",
+    "bolgár": "Bulgarian",
+    "kínai": "Chinese",
+    "horvát": "Croatian",
+    "cseh": "Czech",
+    "dán": "Danish",
+    "holland": "Dutch",
+    "angol": "English",
+    "észt": "Estonian",
+    "finn": "Finnish",
+    "francia": "French",
+    "német": "German",
+    "görög": "Greek",
+    "héber": "Hebrew",
+    "hindi": "Hindi",
+    "magyar": "Hungarian",
+    "olasz": "Italian",
+    "japán": "Japanese",
+    "koreai": "Korean",
+    "lett": "Latvian",
+    "litván": "Lithuanian",
+    "macedón": "Macedonian",
+    "norvég": "Norwegian",
+    "lengyel": "Polish",
+    "portugál": "Portuguese",
+    "román": "Romanian",
+    "orosz": "Russian",
+    "szerb": "Serbian",
+    "szlovák": "Slovak",
+    "szlovén": "Slovenian",
+    "spanyol": "Spanish",
+    "svéd": "Swedish",
+    "török": "Turkish",
+}
+
+EPISODE_REGEXP = re.compile(r'S?(?P<season>\d+)([x_-]|\.)*E?(?P<episode>\d+)', re.IGNORECASE)
+
+def recreate_dir(path):
```

Comment:
Since this is only used for the temp dir can you re-write this in a way that it receives no arguments and specifically processes that folder only?